Wikipedia/UI-V5/WMFArticleListDataSource.h

```diff
@@ -17,9 +18,13 @@ NS_ASSUME_NONNULL_BEGIN
 
 - (MWKHistoryDiscoveryMethod)discoveryMethod;
 
+- (void)setSavedPageList:(MWKSavedPageList*)savedPageList;
+
 @optional
 - (void)deleteArticleAtIndexPath:(NSIndexPath*)indexPath;
 
+- (CGFloat)estimatedItemHeight;
```

Comment:
Did you consider putting this on the Cell class instead? Seems more view related and tied directly to the cell (and delegate methods as opposed to datasource methods).